src/bindings/python/src/openvino/runtime/opset15/__init__.py

```diff
@@ -6,4 +6,5 @@
 
 # TODO (ticket 138273): Add previous opset operators at the end of opset15 development
 from openvino.runtime.opset1.ops import parameter
+from openvino.runtime.opset15.ops import col2im
```

Comment:
do we need `from openvino.runtime.opset1.ops import parameter` above?

Reply:
I'm not certain, my guess is that it was needed in `opset15` for now for the tests to use it. One way or another since this PR is targeted for Code Freeze let's come back to it another time.

Reply:
when you import opset15, all operation from previous opset are automatically imported. Probably that one line was added when opset15 placeholder was being added.